README.md

```diff
@@ -5,38 +5,46 @@
 
 > Rust & RIOT combined for ergonomic embedded development
 
-This is an experimental project to provide a nice base OS for embedded
-development on low-end IoT devices (with some kilobytes of RAM/flash, think
-Cortex-M). It combines the awesome Rust embedded ecosystem with
-[RIOT](https://github.com/RIOT-OS/RIOT).
+RIOT-rs is a project aiming to provide an adequate 
+operating system for cybersecure, memory-safe, low-power IoT,
+based on Rust from the ground up, and formal verification for critical modules. 
+For a more verbose rant, see this
+[manifesto](https://future-proof-iot.github.io/RIOT-rs/dev/manifesto.html).
 
-**This is highly experimental. Expect heavy changes and breakage!**
+Overall, RIOT-rs aims for a 'batteries-included' experience, on par
+with [RIOT](https://github.com/RIOT-OS/RIOT): 
+all the needed OS facilities and a high level of integration for 
+the various libs, tools and toolchains required by low-power IoT app
+development. Targets include various hardware based on 
+32-bit microcontroller architectures (such as Arm Cortex-M, RISC-V, ESP32).
```

Comment:
Drop "Arm"? and, ESP32 is not an architecture per se

Reply:
updated ;)